src/app/api/routes/alerts.py

```diff
@@ -81,15 +81,18 @@ async def create_alert_from_device(
     Below, click on "Schema" for more detailed information about arguments
     or "Example Value" to get a concrete idea of arguments
     """
-
     return await create_alert(AlertIn(**payload.dict(), device_id=device.id), background_tasks)
 
 
 @router.get("/{alert_id}/", response_model=AlertOut, summary="Get information about a specific alert")
-async def get_alert(alert_id: int = Path(..., gt=0), _=Security(get_current_access, scopes=[AccessType.admin])):
+async def get_alert(alert_id: int = Path(..., gt=0),
+                    requester=Security(get_current_access,
+                    scopes=[AccessType.admin, AccessType.user])):
```

Comment:
Let's keep black formatting when we have several lines:
```
async def get_alert(
    alert_id: int = Path(..., gt=0),
    requester=Security(get_current_access, _=[AccessType.admin, AccessType.user])
 ):
```

Reply:
Not sure I get it, didn't you ask in another review to have several lines in those cases?

Reply:
Sure, but 1 line per function argument
Besides, I think we can annotation typing to `requester`